bahmni-lite/docker-compose.yml

```diff
@@ -152,6 +152,80 @@ services:
     ports:
       - '33006:3306'
 
+  metabasedb:
+    image: postgres:${METABASE_POSTGRES_IMAGE_TAG:?}
+    restart: on-failure:[3]
+    profiles: ["metabase"]
+    environment:
+      POSTGRES_DB: ${METABASE_DB_NAME:?}
+      POSTGRES_USER: ${METABASE_DB_USER:?}
+      POSTGRES_PASSWORD: ${METABASE_DB_PASSWORD:?}
+    volumes:
+      - 'metabase-data:/var/lib/postgresql/data'
```

Comment:
Add metabase-data volume in Volumes section too.

Reply:
Added. Thanks !!